### ISSUE TITLE
Structural metadata in CSV

### ISSUE THREAD

**Issue by dosse** [sdmx-twg/sdmx-csv#26]
To allow full processing of SDMX structures in tabular format, e.g. in Excel

**Comment by pedroacarranza** [comment]
@dosse is this still a requirement? I suppose it originated from the .Stat suite?

**Comment by dosse** [comment]
No, this was not a specific need for .Stat Suite. It was especially asked by @egreising, and the most important use case was easy management of item schemes (e.g., using Excel).

**Comment by pedroacarranza** [comment]
TWG agreed to close ticket - This is related to tooling/implementation.
